pymc/tests/test_distributions_timeseries.py

```diff
@@ -147,33 +130,37 @@ def test_gaussianrandomwalk_inference(self):
     @pytest.mark.parametrize("init", [None, pm.Normal.dist()])
     def test_gaussian_random_walk_init_dist_shape(self, init):
         """Test that init_dist is properly resized"""
+
+        def get_init_dist(grw_dist):
+            return grw_dist.owner.inputs[0].owner.inputs[1].owner.inputs[0]
+
         grw = pm.GaussianRandomWalk.dist(mu=0, sigma=1, steps=1, init_dist=init)
-        assert tuple(grw.owner.inputs[-2].shape.eval()) == ()
+        assert tuple(get_init_dist(grw).shape.eval()) == ()
 
         grw = pm.GaussianRandomWalk.dist(mu=0, sigma=1, steps=1, init_dist=init, size=(5,))
-        assert tuple(grw.owner.inputs[-2].shape.eval()) == (5,)
+        assert tuple(get_init_dist(grw).shape.eval()) == (5,)
 
         grw = pm.GaussianRandomWalk.dist(mu=0, sigma=1, steps=1, init_dist=init, shape=2)
-        assert tuple(grw.owner.inputs[-2].shape.eval()) == ()
+        assert tuple(get_init_dist(grw).shape.eval()) == ()
 
         grw = pm.GaussianRandomWalk.dist(mu=0, sigma=1, steps=1, init_dist=init, shape=(5, 2))
-        assert tuple(grw.owner.inputs[-2].shape.eval()) == (5,)
+        assert tuple(get_init_dist(grw).shape.eval()) == (5,)
 
         grw = pm.GaussianRandomWalk.dist(mu=[0, 0], sigma=1, steps=1, init_dist=init)
-        assert tuple(grw.owner.inputs[-2].shape.eval()) == (2,)
+        assert tuple(get_init_dist(grw).shape.eval()) == (2,)
 
         grw = pm.GaussianRandomWalk.dist(mu=0, sigma=[1, 1], steps=1, init_dist=init)
-        assert tuple(grw.owner.inputs[-2].shape.eval()) == (2,)
+        assert tuple(get_init_dist(grw).shape.eval()) == (2,)
 
         grw = pm.GaussianRandomWalk.dist(mu=np.zeros((3, 1)), sigma=[1, 1], steps=1, init_dist=init)
-        assert tuple(grw.owner.inputs[-2].shape.eval()) == (3, 2)
+        assert tuple(get_init_dist(grw).shape.eval()) == (3, 2)
 
     def test_shape_ellipsis(self):
         grw = pm.GaussianRandomWalk.dist(
             mu=0, sigma=1, steps=5, init_dist=pm.Normal.dist(), shape=(3, ...)
         )
         assert tuple(grw.shape.eval()) == (3, 6)
```

Comment:
I am getting `grw.shape.eval() = (3, 6, 6)` with respect to the `change_size` implemented above

Reply:
Done